### ISSUE TITLE
Support cards in multiple languages.

### ISSUE THREAD

**Issue by phulin** [dekkerglen/CubeCobra#975]
This needs only to include image and translated name

**Comment by ruler501** [comment]
This will likely require #970 to correlate cards.

**Comment by dekkerglen** [comment]
Not necessarily, the names and images could just be properties of the existing objects we have.

**Comment by ruler501** [comment]
How would you handle pricing them differently in that case?

**Comment by dekkerglen** [comment]
I don't see why we couldn't handle it the same way

**Comment by dekkerglen** [comment]
This way we could also add a field in the context modal for language, instead of bloating the versions drop down
